roles/check-swlib/tasks/main.yml

```diff
@@ -62,7 +62,7 @@
   ignore_errors: true
   with_items:
     - "{{ opatch_patches }}"
-  when: item.release == oracle_ver
+  when: item.release == oracle_ver and oracle_rel != "base"
```

Comment:
Clarification sought:

What is the differnce between L48 and the change in L65?

To be consistent with line: https://github.com/google/bms-toolkit/blob/master/roles/check-swlib/tasks/main.yml#L48
```
  when: item.base == oracle_ver and item.release == oracle_rel
```

can we just arrange the proposed change so both are equally skipped from checks with a `--no-patch` invocation.

Reply:
The difference is that `opatch_patches` is set unconditionally (https://github.com/google/bms-toolkit/blob/743971aafde6813b9f9df6be98d2ee9696e335f2/roles/common/defaults/main.yml#L220) but that `gi_patches` and `rdbms_patches` are only set if there are patches to apply (or, in other words, if it's not the base release).